scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -63,12 +80,25 @@ func on_jump_input() -> void:
 
 func on_jump_input_cancelled() -> void:
 	_is_jump_button_held = false
+	
+func _on_entered_climbable() -> void:
+	_is_climbing = true
+	_can_climb = true
+	print("Entered climbable area")
+
+func _on_exited_climbable() -> void:
+	_is_climbing = false
+	_can_climb = false
+	if _is_up_pressed:
+		velocity.y = -LAUNCH_VELOCITY
+	print("Exited climbable area")
+
```

Comment:
these should be in the base. Anything can enter a climbable area

Reply:
Moved to base_movement_component